pkg/game/elements/placed_tile.go

```diff
@@ -93,3 +129,12 @@ type PlacedTile struct {
 func NewStartingTile(tileSet tilesets.TileSet) PlacedTile {
 	return ToPlacedTile(tileSet.StartingTile)
 }
+
+func (placement *PlacedTile) GetPlacedFeatureAtSide(sideToCheck sideMod.Side) *PlacedFeature {
+	for i, feature := range placement.TileWithMeeple.Features {
+		if sideToCheck&feature.Sides == sideToCheck {
+			return &placement.TileWithMeeple.Features[i]
+		}
+	}
+	return nil
+}
```

Comment:
Will this behave correctly with tiles that have more than one feature on one side?
For example, monastery with a single road has both a road and a field on one side

Reply:
you're right, the feature type also needs to be passed